pajbot/apiwrappers/twitchemotesapi.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from requests import HTTPError
+
+from pajbot.apiwrappers.response_cache import TwitchChannelEmotesSerializer
+from pajbot.apiwrappers.base import BaseAPI
+
+log = logging.getLogger(__name__)
+
+
+class TwitchEmotesAPI(BaseAPI):
+    def __init__(self, redis):
+        super().__init__(base_url="https://api.twitchemotes.com/api/v4/", redis=redis)
+
+    def fetch_channel_emotes(self, channel_id, channel_name):
+        """Returns a tuple of three lists of emotes, each one corresponding to tier 1, tier 2 and tier 3 respectively.
+        Tier 2 and Tier 3 ONLY contain the respective extra emotes added to that tier, typically tier 2 and tier 3
+        will contain exactly one or zero emotes."""
+        # circular import prevention
+        from pajbot.managers.emote import EmoteManager
+
+        try:
+            resp = self.get(["channels", channel_id])
+            plans = resp["plans"]
+            if len(plans) <= 0:
+                log.warning(f"No subscription plans found for channel {channel_name}")
+                return [], [], []
+
+            # plans["$4.99"] is tier 1
+            # plans["$9.99"] is tier 2
+            # plans["$24.99"] is tier 1
+            ret_data = [[], [], []]
```

Comment:
Just nitpicking but can‘t we declare this as `([], [], [])` to save the call to `tuple()` later?